tools/mount_gcsfuse/main.go

```diff
@@ -81,12 +81,15 @@ func makeGcsfuseArgs(
 
 		// Special case: support mount-like formatting for gcsfuse bool flags.
 		case "implicit_dirs",
-			"foreground",
 			"experimental_local_file_cache",
 			"enable_storage_client_library",
 			"reuse_token_from_url",
 			"enable_nonexistent_type_cache":
-			args = append(args, "--"+strings.Replace(name, "_", "-", -1))
+			if value == "" {
```

Comment:
when we will not pass the value, it will take default value(true), when we pass some value it will pass that value.

Reply:
Can we do something like:
// value should be true in case '--flag' & '--flag=true'
if value != "false" {
   value = "true"
}
args = append(args, "--"+strings.Replace(name, "_", "-", -1)+"="+value)

Reply:
Fixed